test-data/unit/check-generics.test

```diff
@@ -2994,7 +2994,7 @@ from typing import TypeVar, Protocol, Generic, Optional
 T = TypeVar('T')
 
 class F(Protocol[T]):
-    def __call__(self, __x: T) -> T: ...
```

Comment:
I personally don't think these should be renamed. These also work fine with `inspect.signature`.

Reply:
I am not sure what you mean:

```python
>>> import inspect
>>> class C:
...     def f(self, __x: int) -> str: ...
>>> print(inspect.signature(C.f))
(self, _C__x: int) -> str
>>> print(inspect.signature(C().f))
(_C__x: int) -> str
```

Reply:
Oh wow, sorry that was really my bad.